### DISCUSSION TITLE
Move `Virtual Desktops` of `Task View` to the top

### DISCUSSION THREAD

**Discussion by bilogic** [valinet/ExplorerPatcher#3473] [Ideas]
![image](https://github.com/user-attachments/assets/f85174d4-d6e2-4fb3-b425-396a1d29e3cb)

Possible to move the virtual desktops to the top? This will avoid the need to keep moving the mouse up and down the screen when the taskbar is at the top.

**Comment by pyrates999** [comment]
It depends if the code is already there to do this.  If it is not, then EP has no way to do this.

**Reply by bilogic** [reply]
Windows 10's was always at the top, even when the taskbar is at the bottom, would that mean the code exists?

**Reply by pyrates999** [reply]
So in windows 10, the part outlined in red was always at the top?

**Reply by bilogic** [reply]
yup correct, always on top.

i suppose in a bid to show they are not copying from MacOS, they moved it down in W11

**Reply by pyrates999** [reply]
Oh so in windows 11, it's at the bottom?  If so, then the code could exist where it is at the top.  But as we are seeing from windows 11 24h2, code used in windows 10 is slowly being removed.

**Reply by bilogic** [reply]
My impression of this project was it adds in the code to make whatever changes necessary.. that would minimize breaks from microsoft's actions.. apparently that was a wrong impression
